app/controllers/admin/students_controller.rb

```diff
@@ -1,20 +0,0 @@
-module Admin
-  class StudentsController < ResourceController
```

Comment:
fully deleting this controller will not allow these views: https://github.com/rootselementary/check_in_tool_rails/tree/c5cd0a4d16369362311b3cfdaba3e62a13e94067/app/views/admin/students to be rendered or that part of the app from handling crud tasks on students. is that what you intend? Maybe so, since no tests are broken, I can only assume that if the student crud actions were implemented, then someone wrote tests around it...?